<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>725</width>
         <height>466</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="labPic">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>目录树</string>
    </property>
    <addaction name="actAddCatalog"/>
    <addaction name="actAddFile"/>
    <addaction name="actDeleteNode"/>
    <addaction name="actTraverseNode"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="actAMP"/>
    <addaction name="actRED"/>
    <addaction name="actCurrentAMP_RDE"/>
    <addaction name="actWide"/>
    <addaction name="actHeight"/>
    <addaction name="actWindowsFloat"/>
    <addaction name="actWindowsShow"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actAddCatalog"/>
   <addaction name="actAddFile"/>
   <addaction name="actDeleteNode"/>
   <addaction name="actTraverseNode"/>
   <addaction name="separator"/>
   <addaction name="actAMP"/>
   <addaction name="actRED"/>
   <addaction name="actCurrentAMP_RDE"/>
   <addaction name="actWide"/>
   <addaction name="actHeight"/>
   <addaction name="separator"/>
   <addaction name="actWindowsFloat"/>
   <addaction name="actWindowsShow"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>目录</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>节点类型</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actAddCatalog">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/20.png</normaloff>:/icon/20.png</iconset>
   </property>
   <property name="text">
    <string>添加目录</string>
   </property>
   <property name="toolTip">
    <string>添加目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAddFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/19.png</normaloff>:/icon/19.png</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="toolTip">
    <string>添加文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actDeleteNode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/10.png</normaloff>:/icon/10.png</iconset>
   </property>
   <property name="text">
    <string>删除节点</string>
   </property>
   <property name="toolTip">
    <string>删除节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actTraverseNode">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/8.png</normaloff>:/icon/8.png</iconset>
   </property>
   <property name="text">
    <string>遍历节点</string>
   </property>
   <property name="toolTip">
    <string>遍历节点</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAMP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/4.png</normaloff>:/icon/4.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actRED">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/16.png</normaloff>:/icon/16.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actCurrentAMP_RDE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/1.png</normaloff>:/icon/1.png</iconset>
   </property>
   <property name="text">
    <string>实际大小</string>
   </property>
   <property name="toolTip">
    <string>实际大小</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actWide">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/13.png</normaloff>:/icon/13.png</iconset>
   </property>
   <property name="text">
    <string>合适宽度</string>
   </property>
   <property name="toolTip">
    <string>合适宽度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actHeight">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/23.png</normaloff>:/icon/23.png</iconset>
   </property>
   <property name="text">
    <string>合适高度</string>
   </property>
   <property name="toolTip">
    <string>合适高度</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actWindowsFloat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/15.png</normaloff>:/icon/15.png</iconset>
   </property>
   <property name="text">
    <string>窗口浮动</string>
   </property>
   <property name="toolTip">
    <string>窗口浮动</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actWindowsShow">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/9.png</normaloff>:/icon/9.png</iconset>
   </property>
   <property name="text">
    <string>窗口可见</string>
   </property>
   <property name="toolTip">
    <string>窗口可见</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/17.png</normaloff>:/icon/17.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
